src/modules/package.rs

```diff
@@ -54,6 +53,16 @@ fn extract_cargo_version(file_contents: &str) -> Option<String> {
     Some(formatted_version)
 }
 
+fn extract_nimble_version() -> Option<String> {
+    let cmd_output = utils::exec_cmd("nimble", &["dump", "--json"], Duration::from_millis(500))?;
```

Comment:
Can you change this to use `context.exec_cmd` so this uses the global timeout?

Reply:
Yeah, I've already changed it. Sorry about that. I was trying to push it along with tests. I will work on it sometime later.